### DISCUSSION TITLE
Please add new protocols

### DISCUSSION THREAD

**Discussion by kapybarovv** [Gozargah/Marzban#1261] [Ideas]
Hello! Please add more VPN protocols such as: L2TP+IPsec or IKEv2

**Comment by M03ED** [comment]
these are not good protocols for using in a country like Iran or China, they gonna get blocked so fast, also marzban is completely based on xray-core , if xray add them we can add them too but i don't this this happen ever

**Reply by kapybarovv** [reply]
Unfortunately, most routers only support these protocols: (

**Reply by M03ED** [reply]
also most routers support open-wrt
search about it
